packages/pipeline-editor/src/PipelineEditorWidget.tsx

```diff
@@ -1048,7 +1051,13 @@ export class PipelineEditor extends React.Component<
       return;
     }
 
-    const runtimes = await PipelineService.getRuntimes();
+    const runtimes = await PipelineService.getRuntimes(false);
+    const local_runtime: any = {
+      name: 'local',
+      display_name: 'Run in-place locally'
```

Comment:
'in-place locally' is redundant and one of the two should be picked.

Reply:
though English-wise I agree, in this case they clarify different things. "in-place" tells the user that the notebooks are being run where they currently are, and "locally" tells them that it's not sending the job to a remote service. If you remove one it changes the meaning, just "locally" could imply that it's running locally but using a local instance of a service, and just "in-place" is (at least to me) not clear enough on it's own for a user to know where it will run.

Reply:
I do believe  the current description is too ambiguous. "Local" could refer to the machine where the web browser is running or the machine where JupterLab is running.